README.md

```diff
@@ -120,6 +120,62 @@ user.
 - [html2text](https://pypi.python.org/pypi/html2text)
 - [in_place](https://pypi.org/project/in-place/) (only used by newfeed.py; otherwise optional)
 
+# Docker Usage
```

Comment:
Could this please go into DOCKER.md instead of into README.md?

Reply:
Isn't it easier to read if it's in the main README ? Or make a summary for an easier understand and/or redirect to the DOCKER.md